pkgs/development/libraries/librdf/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, fetchurl, pkgconfig, autoreconfHook
 , librdf_raptor2, ladspaH, openssl, zlib
+, doCheck ? stdenv.config.doCheckByDefault or false, ladspaPlugins
```

Comment:
I don't really like having this here. Would it be possible to avoid the conditional and always patch like we are potentially going to run the tests?